vault/resource_pki_secret_backend_config_ca_test.go

```diff
@@ -41,9 +43,9 @@ resource "vault_mount" "test" {
 }
 
 resource "vault_pki_secret_backend_config_ca" "test" {
-  depends_on = [ "vault_mount.test" ]
-  backend = vault_mount.test.path
-  pem_bundle = <<EOT
+  depends_on       = ["vault_mount.test"]
```

Comment:
Since backend references vault_mount.test there's no need to declare the extra dependency here.

```suggestion
```